storageserver/src/apps/storaged/storage.cpp

```diff
@@ -198,7 +198,7 @@ int StorageApp::Main()
     LOG(debug, "Server was attempted stopped, shutting down");
     // Create guard that will forcifully kill storage if destruction takes longer
     // time than given timeout.
-    vespalib::ShutdownGuard shutdownGuard(std::chrono::milliseconds(_maxShutdownTime));
+    vespalib::ShutdownGuard shutdownGuard((std::chrono::milliseconds(_maxShutdownTime)));
```

Comment:
Why, what is the ambiguity ?

Reply:
[100%] Building CXX object storageserver/src/apps/storaged/CMakeFiles/storageserver_storaged_app.dir/storage.cpp.o
/home/tegge/git/vespa/storageserver/src/apps/storaged/storage.cpp:201:42: error: parentheses were disambiguated as a function declaration [-Werror,-Wvexing-parse]
    vespalib::ShutdownGuard shutdownGuard(std::chrono::milliseconds(_maxShutdownTime));
                                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/tegge/git/vespa/storageserver/src/apps/storaged/storage.cpp:201:43: note: add a pair of parentheses to declare a variable
    vespalib::ShutdownGuard shutdownGuard(std::chrono::milliseconds(_maxShutdownTime));
                                          ^
                                          (                                          )
1 error generated.